### ISSUE TITLE
Variable initialized with parentheses highlighted as function in C++ 

### ISSUE THREAD

**Issue by Craie** [pygments/pygments#2035]
A variable initialized using parentheses in C++ is highlighted as a function rather than a variable:

    long var2(42'012'593L);

This highlights var2 as blue in default style, for instance.  Other variables (= or {} initialized) are fine.

Sorry about the earlier mixup.  I upgraded, deleted my whole minted cache, and rebuilt and all is well with the quotes now!  Thanks for the pointer!

**Comment by birkenfeld** [comment]
Yes, this one is a bit problematic... Since the C family declaration syntax is so complex and not using keywords, there's basically no chance to categorize these correctly with the lexing we do. I don't have high hopes for this to be fixed...
